instrumentation/src/main/java/io/opentelemetry/android/config/OtelRumConfig.java

```diff
@@ -24,6 +24,7 @@ public class OtelRumConfig {
     private DiskBufferingConfiguration diskBufferingConfiguration =
             DiskBufferingConfiguration.builder().build();
     private boolean networkChangeMonitoringEnabled = true;
+    private boolean debugLogEnabled = false;
```

Comment:
If I look at this option name without knowing anything about this repo, my first impression of it would be that it's related to turning on/off the internal logs that are produced by this lib. Maybe if I wasn't too knowledgeable about OTel in general, I might even think that it might have something to do with OTel log signals... In any case, I think the name doesn't convey that it's related to logging Spans at the time of exporting.

Reply:
Yeah that's a good call out. I think a more precise name is in order -- maybe something like `debugSpanLoggingEnabled` or similar.